njsscan/__main__.py

```diff
@@ -126,16 +188,25 @@ def main():
                         action='store_true')
     args = parser.parse_args()
     if args.path:
+        show_progress = False
+        if args.json or args.sonarqube:
+            show_progress = True
+
         scan_results = NJSScan(
```

Comment:
This API is used by third-party tools. I do not want to break it for them with this change.

Reply:
it was my mistake, fixed it!